src/app/brukerprofil/kontonummer/EndreKontonummerForm.tsx

```diff
@@ -0,0 +1,217 @@
+///<reference path="kontonummerUtils.ts"/>
+import * as React from 'react';
+import { FormEvent } from 'react';
+import { Action } from 'history';
+import { connect, Dispatch } from 'react-redux';
+
+import Input from 'nav-frontend-skjema/lib/input';
+
+import { STATUS } from '../../../redux/utils';
+import { BankAdresse, Person } from '../../../models/person/person';
+import { AppState } from '../../../redux/reducer';
+import { VeilederRoller } from '../../../models/veilederRoller';
+import Undertittel from 'nav-frontend-typografi/lib/undertittel';
+import { FormKnapperWrapper } from '../BrukerprofilForm';
+import KnappBase from 'nav-frontend-knapper';
+import RadioPanelGruppe from 'nav-frontend-skjema/lib/radio-panel-gruppe';
+import { ChangeEvent } from 'react';
+import {
+    removeWhitespaceAndDot,
+    formaterNorskKontonummer,
+    validerKontonummer,
+    erBrukersKontonummerUtenlandsk,
+    BankKontoUtenOptionals,
+    tomBankKonto
+}
+    from './kontonummerUtils';
+import UtenlandskKontonrInputs from './UtenlandskKontonummerInputs';
+import RequestTilbakemelding from '../RequestTilbakemelding';
+import { Kodeverk } from '../../../models/kodeverk';
+
+enum bankEnum {
+    erNorsk = 'Kontonummer i Norge',
+    erUtenlandsk = 'Kontonummer i utlandet'
+}
+
+const radioKnappProps = [
+    {
+        label: bankEnum.erNorsk,
+        value: bankEnum.erNorsk
+    },
+    {
+        label: bankEnum.erUtenlandsk,
+        value: bankEnum.erUtenlandsk
+    }
+];
+
+interface State {
+    bankKontoInput: BankKontoUtenOptionals;
+    norskKontoRadio: boolean;
+}
+
+interface DispatchProps {
+}
+
+interface StateProps {
+    status: STATUS;
+}
+
+interface OwnProps {
+    person: Person;
+    veilederRoller?: VeilederRoller;
```

Comment:
Denne trenger ikke være optional lenger :)

Reply:
( Hvis du har merget inn master )

Reply:
aha! trenger jeg egentlig veilderroller her? trenger man rolle for å endre kontonummer?